### PR TITLE
Fixed #27324 -- Simplified get_constraints on Oracle.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Comment by charettes** [comment]
Buildbot, test on oracle.

**Comment by timgraham** [comment]
buildbot, test on oracle.

File: django/db/backends/oracle/introspection.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def get_constraints(self, cursor, table_name):
         Retrieves any constraints or keys (unique, pk, fk, check, index) across one or more columns.
         """
         constraints = {}
-        # Loop over the constraints, getting PKs and uniques
+        # Loop over the constraints, getting PKs, uniques and checks
```
Comment (timgraham) [inline django/db/backends/oracle/introspection.py]: , and

File: django/db/backends/oracle/introspection.py [hunk 2]
```diff
@@ -167,29 +167,32 @@ def get_constraints(self, cursor, table_name):
                     WHEN 'P' THEN 1
                     ELSE 0
                 END AS is_primary_key,
-                CASE user_indexes.uniqueness
-                    WHEN 'UNIQUE' THEN 1
+                CASE
+                    WHEN EXISTS (SELECT 1
```
Comment (timgraham) [inline django/db/backends/oracle/introspection.py]: I'd use hanging indent as we do for Python code:
```
WHEN EXISTS (
    SELECT 1
    FROM user_indexes
    WHERE user_indexes.index_name = user_constraints.index_name
    AND user_indexes.uniqueness = 'UNIQUE'
)
```

**Pull request merged**